pkg/install/installer.go

```diff
@@ -374,44 +468,36 @@ func findProjectDir() (string, error) {
 	return projectDir, nil
 }
 
-// copyCMakeIntegration copies the CMake integration to destDir and returns the
-// path that should be registered with the CMake package registry, if needed on
-// the platform.
-// Directories are created as needed.
-func (i *Installer) copyCMakeIntegration(destDir string) (string, error) {
-	cmakeSrc := filepath.Join(i.projectDir, "tools", "cmake", "cifuzz")
-	opts := copy.Options{
-		// Skip copying the replayer, which is a symlink on UNIX but checked out
-		// by git as a file containing the relative path on Windows. It is
-		// handled below.
-		OnSymlink: func(string) copy.SymlinkAction {
-			return copy.Skip
-		},
-	}
-	err := copy.Copy(cmakeSrc, destDir, opts)
-	if err != nil {
-		return "", errors.WithStack(err)
-	}
+func validateTargetDir(installDir string) (string, error) {
+	var err error
 
-	// Copy the replayer, which is a symlink and thus may not have been copied
-	// correctly on Windows.
-	replayerSrc := filepath.Join(i.projectDir, "tools", "replayer", "src", "replayer.c")
-	replayerDir := filepath.Join(destDir, "src")
-	err = os.MkdirAll(replayerDir, 0755)
-	if err != nil {
-		return "", errors.WithStack(err)
-	}
-	err = copy.Copy(replayerSrc, filepath.Join(replayerDir, "replayer.c"))
-	if err != nil {
-		return "", errors.WithStack(err)
+	if strings.HasPrefix(installDir, "~") {
+		home, err := os.UserHomeDir()
+		if err != nil {
+			return "", errors.WithStack(err)
+		}
+		installDir = home + strings.TrimPrefix(installDir, "~")
 	}
-	err = copy.Copy(replayerSrc, filepath.Join(replayerDir, "replayer.cpp"))
-	if err != nil {
-		return "", errors.WithStack(err)
+
+	if installDir == "" {
+		installDir, err = os.MkdirTemp("", "cifuzz-install-dir-")
+		if err != nil {
+			return "", errors.WithStack(err)
+		}
+	} else {
+		installDir, err = filepath.Abs(installDir)
+		if err != nil {
+			return "", errors.WithStack(err)
+		}
+
+		exists, err := fileutil.Exists(installDir)
+		if err != nil {
+			return "", errors.WithStack(err)
```

Comment:
Duplicate stack trace
```suggestion
			return "", err
```